project-template/src/main/java/edu/hw1/Main.java

```diff
@@ -11,17 +13,311 @@ public final class Main {
     private Main() {
     }
 
-    public static void main(String[] args) {
-        // Press Alt+Enter with your caret at the highlighted text to see how
-        // IntelliJ IDEA suggests fixing it.
-        LOGGER.info("Hello and welcome!");
+    //task1
+    public static int minutesToSeconds(String length) {
+        String[] parts = length.split(":");
+        int minutes = Integer.parseInt(parts[0]);
+        int seconds = Integer.parseInt(parts[1]);
+        if (seconds >= 60){
+            return -1;
+        }
+        int totalSeconds = minutes * 60 + seconds;
+        return totalSeconds;
+    }
+
+    //task2
+    public static int countDigits(int number) {
+        int count = 0;
+        if(number == 0 ){
+            count++;
+            return count;
+        } else {
```

Comment:
`Enhancement` давай не будем писать else после return
```suggestion
        } 
```